### PR TITLE
Update UMS.conf

### PR DESCRIPTION
**Pull request by Sami32**
Fix one missing line.

---
This change is [Reviewable](https://reviewable.io/reviews/universalmediaserver/universalmediaserver/956)

### CODE REVIEW

File: src/main/external-resources/UMS.conf [hunk 1]
```diff
@@ -1410,6 +1410,7 @@ logging_filter_console =
 # get back things filtered out by Log Level above, but you can further filter
 # out more before it's written to the log window. Valid values are:
```
Comment (SubJunk) [inline src/main/external-resources/UMS.conf]: @Sami32 it seems there's some redundancy on this line too, can you clean that up while you're at it?

**Comment by Sami32** [comment]
@SubJunk Right :+1:

**Comment by SubJunk** [comment]
@Sami32 sure, comments would be useful

**Comment by Sami32** [comment]
@SubJunk Done.

**Pull request merged**